test/FeeTaker.js

```diff
@@ -0,0 +1,167 @@
+const hre = require('hardhat');
+const { ethers } = hre;
+const { expect } = require('@1inch/solidity-utils');
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
+const { deploySwapTokens } = require('./helpers/fixtures');
+const { buildOrder, buildTakerTraits, signOrder } = require('./helpers/orderUtils');
+const { ether, trim0x } = require('./helpers/utils');
+
+describe('FeeTaker', function () {
+    let addr, addr1, addr2, addr3;
```

Comment:
Let's move definition to fixture?

Reply:
What would be the benefit of doing so?

Reply:
It was decided to refactor all tests together as part of another task.